pkgs/development/python-modules/pykdtree/default.nix

```diff
@@ -24,6 +24,10 @@ buildPythonPackage rec {
     hash = "sha256-zCCypnxkBWSFoxTSwrbbo1SvfuHI+42uG+byk2o3Q0E=";
   };
 
+  postPatch = ''
+    substituteInPlace pyproject.toml --replace "numpy>=2.0.0rc1,<3" "numpy"
```

Comment:
is there a specific reason this is used over `pythonRelaxDepsHook`?

Reply:
`pythonRelaxDepsHook` doesn't apply to build-system modules, i believe.